client/data/transactions/hooks.ts

```diff
@@ -41,6 +41,7 @@ export interface Transaction {
 	// A field to identify the payment's source.
 	// Usually last 4 digits for card payments, bank name for bank transfers...
 	source_identifier: string;
+	source_device?: null | string;
```

Comment:
`source_device` is declared as a nullable property. Do we need to explicitly add `null` to its list of possible values?